apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Snippets/UrlSignerSnippets.cs

```diff
@@ -63,15 +63,15 @@ public UrlSignerSnippets(StorageSnippetFixture fixture)
         }
 
         [Fact]
-        public async void SignedURLPut()
+        public async Task SignedURLPut()
         {
             var bucketName = _fixture.BucketName;
-            var credentialsFilePath = GetCredentialsFilePath();
+            var credential = (await GoogleCredential.GetApplicationDefaultAsync()).UnderlyingCredential as ServiceAccountCredential;
             var httpClient = new HttpClient();
 
             // Sample: SignedURLPut
             // Create a signed URL which allows the requester to PUT data with the text/plain content-type.
-            UrlSigner urlSigner = UrlSigner.FromServiceAccountPath(credentialsFilePath);
+            UrlSigner urlSigner = UrlSigner.FromServiceAccountCredential(credential);
```

Comment:
So we can still use FromServiceAccountPath if we want to, right? My guess is that that will be a very common scenario, as users may have no other reason to get the credential directly.